src/routes/home/+page.svelte

```diff
@@ -0,0 +1,225 @@
+					typography({ variant: 'body-medium' }),
+					'font-semibold px-9 py-2 leading-5',
+					'max-md:text-xs max-md:leading-3 max-md:px-3 max-md:py-1.5'
+				)}
+			>
+				รู้จักกับสโมสรนิติฯ
+			</Button>
+		</div>
+	</div>
+
+	<div
+		class="flex w-full h-[642px] py-[72px] px-[120px] gap-12 items-center text-sucu-gray-dark bg-white max-md:hidden"
+	>
+		<div
+			class="flex flex-col w-[297px] h-[208px] min-w-[297px] min-h-[208px] justify-between text-end"
+		>
+			<div class={cn(typography({ variant: 'heading1' }), 'tracking-tight leading-none')}>
+				ประกาศ
+			</div>
+			<div class={cn(typography({ variant: 'body-medium' }), 'tracking-tight leading-7')}>
+				ประกาศล่าสุดของสโมสรนิสิตจุฬาฯ อบจ. และสภานิสิตจุฬาฯ
+			</div>
+			<div class="flex justify-end">
+				<Button
+					size="sm"
+					class={cn(
+						typography({ variant: 'body-medium' }),
+						'font-semibold px-9 py-2 leading-5 bg-sucu-pink-01 hover:bg-sucu-pink-02',
+						'max-md:text-xs max-md:leading-3 max-md:px-3 max-md:py-1.5'
+					)}
+				>
+					ดูประกาศเพิ่มเติม
+				</Button>
+			</div>
+		</div>
+		<div
+			class="flex overflow-auto p-6 gap-6 scrollbar-thin scrollbar-thumb-sucu-gray-light scrollbar-track-white"
+		>
+			{#each announcementCard as card}
+				<AnnouncementCard
+					imageURL={card.imageURL}
+					title={card.title}
+					createdAt={card.createdAt}
+					createdBy={card.createdBy}
+					linkHref={card.linkHref}
+				/>
+			{/each}
+		</div>
+	</div>
+	<div class="flex flex-col h-[483px] p-[26px] gap-4 text-sucu-gray-dark bg-white md:hidden">
+		<div class="flex flex-col h-[76px] gap-3">
+			<div class={cn(typography({ variant: 'heading2' }), 'tracking-tight leading-none')}>
+				ประกาศ
+			</div>
+			<div class={cn(typography({ variant: 'body-normal' }), 'tracking-tight leading-none')}>
+				ประกาศล่าสุดของสโมสรนิสิตจุฬาฯ อบจ. และสภานิสิตจุฬาฯ
+			</div>
+		</div>
+		<div
+			bind:this={container}
+			class="flex h-[257px] px-2.5 gap-2.5 overflow-x-auto items-center scrollbar-none snap-x snap-proximity"
+		>
+			{#each announcementCard as card}
+				<AnnouncementCard
+					imageURL={card.imageURL}
+					title={card.title}
+					createdAt={card.createdAt}
+					createdBy={card.createdBy}
+					linkHref={card.linkHref}
+					className="snap-start"
+				/>
+			{/each}
+		</div>
+		<div class="flex justify-center gap-2">
+			{#each announcementCard as _, index}
+				<div
+					class={`w-[5px] h-[5px] rounded-full ${index === activeSlide ? 'bg-sucu-gray-dark' : 'bg-sucu-gray'}`}
+				></div>
+			{/each}
+		</div>
+		<div class="flex justify-center">
+			<Button
+				size="sm"
+				class={cn(
+					typography({ variant: 'body-normal' }),
+					'font-semibold px-3 py-1.5 leading-6 bg-sucu-pink-01 hover:bg-sucu-pink-02'
+				)}
+			>
+				ดูประกาศทั้งหมด
+			</Button>
+		</div>
+	</div>
+
+	<img src={sucuHomeBanner2} alt="banner" class="w-full h-auto bg-cover max-md:hidden" />
+	<img src={sucuHomeBanner3} alt="banner" class="w-full h-auto bg-cover md:hidden" />
```

Comment:
อาจจะ group อันนี้เป็นอีก component นึง
```suggestion
  <Banner banner={sucuHomeBanner2}  bannerMobile={sucuHomeBanner3} />
```

Reply:
<img width="296" alt="Screenshot 2567-10-17 at 11 15 17" src="https://github.com/user-attachments/assets/7eae5105-92c9-47f1-b88e-b9173aebe8ad">

Reply:
เอาเป็นว่าเป็นแบบนี้ไปก่อนเดี่ยว group เป็น component อีกอันนึงเพราะว่าอาจจะมี logic ของมันเอง เพิ่ม

Reply:
เครมึง